### PR TITLE
[4.4] [Guided Tours] The categories tour is not visible to Managers

### PR DESCRIPTION
**Pull request by obuisard**
Pull Request for Issue #44089.

### Summary of Changes

When the tour is a category, the authorization to run should come from the extension associated with those categories.

### Testing Instructions

Create a user with Manager privileges.
Login as that user.

### Actual result BEFORE applying this Pull Request

The tour 'How to create a category' does not show in the list of tours (in the modal).

### Expected result AFTER applying this Pull Request

The tour 'How to create a category' shows in the list of tours (in the modal).

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/modules/mod_guidedtours/src/Helper/GuidedToursHelper.php [hunk 1]
```diff
@@ -59,6 +59,9 @@ public function getTours(Registry $params, AdministratorApplication $app)
             $uri = new Uri($item->url);
 
             if ($extension = $uri->getVar('option')) {
+                if ($extension == 'com_categories') {
```
Comment (brianteeman) [inline administrator/modules/mod_guidedtours/src/Helper/GuidedToursHelper.php]: Should it not be a strict comparison
```suggestion
                if ($extension === 'com_categories') {
```

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 86706a88d77eb47ae868ad01e64984e8102110e2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44116">issues.joomla.org/tracker/joomla-cms/44116</a>.</sub>

**Comment by exlemor** [comment]
I have tested this item :white_check_mark: successfully on 86706a88d77eb47ae868ad01e64984e8102110e2<br /><br />Nice fix! I have successfully tested this. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44116">issues.joomla.org/tracker/joomla-cms/44116</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44116">issues.joomla.org/tracker/joomla-cms/44116</a>.</sub>

**Pull request merged**